SitumWayfinding/Classes/Positioning/PositioningViewController.swift

```diff
@@ -512,13 +544,16 @@ class PositioningViewController: UIViewController, GMSMapViewDelegate, UITableVi
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell: UITableViewCell? = tableView.dequeueReusableCell(withIdentifier: "LevelCellIdentifier")
+        cell?.roundCorners(corners: [.topLeft, .topRight, .bottomLeft, .bottomRight], radius: floorSelectorCornerRadius)
         
         if let level: SITFloor = orderedFloors(buildingInfo: buildingInfo)?[indexPath.row] {
             // [06/08/19] This check is here because older buildings without the name field give unexpected nulls casted to string
             let shouldDisplayLevelName = !(level.name.isEmpty || (level.name == "<null>"))
             let textToDisplay: String = shouldDisplayLevelName ? level.name : String(level.floor)
             cell?.textLabel?.text = String(format: "%@", textToDisplay)
-            cell?.backgroundColor = getBackgroundColor(forFloor: level, atRow: indexPath.row)
+            let cellCollors  = getCellColors(forFloor: level, atRow: indexPath.row)
+            cell?.textLabel?.textColor = cellCollors.textColor
+            cell?.backgroundColor = cellCollors.backgroundColor
```

Comment:
O contraste entre estes botóns e o mapa de fondo creo que é demasiado pouco e fai que apenas se distinga o selector. Ademáis no modo claro en android hai unha pequena sombra que fai que este contraste o faga máis legible, igual deberíamos unificar?